### DISCUSSION TITLE
一言难尽的反官方民间润学。还没润就排挤人。华裔最爱欺负华裔么。

### DISCUSSION THREAD

**Discussion by BigDan0** [The-Run-Philosophy-Organization/run#515] [General]
乌烟瘴气。这就是海外华人要面对的环境。可真把人恶心坏了。

怎么感觉你们不让人出国。灌一堆废料，不反华排华还不行。
华人内部搞分裂，搬弄是非。还有人追捧，当个宝。
自封官方就引起不适。哪里来的神经病，民科太祖二世祖。搞什么鬼。
妖魔鬼怪开大会。讨人嫌。

**Comment by ghost** [comment]
"Joined 1 hour ago"

**Reply by BigDan0** [reply]
> "Joined 1 hour ago"

污染时间线。恶心。正常人大号，不愿意掺和。

**Reply by ghost** [reply]
> > "Joined 1 hour ago"
> 
> 污染时间线。恶心。正常人大号，不愿意掺和。

首先这个官方是 bboceng 自封的，然后 tm 他现在又不管这个仓库了（
还有，请指出此仓库反华排华的内容

**Comment by yoloswin** [comment]
嗯？

**Comment by MarionNoel** [comment]
恭喜你答对了，润太祖还真就是个二世祖，它从小就满嘴谎言
https://github.com/MarionNoel/juzhanggongzi
